Fix errors from man pointed out by lintian:

 usr/share/man/man7/pcap-tstamp.7.gz 171: warning [p 2, 11.0i]: cannot adjust line

--- a/pcap-tstamp.manmisc.in
+++ b/pcap-tstamp.manmisc.in
@@ -166,10 +166,17 @@
 stamps in the file.
 .SH SEE ALSO
 pcap_set_tstamp_type(3PCAP),
+.BR
 pcap_list_tstamp_types(3PCAP),
+.BR
 pcap_tstamp_type_val_to_name(3PCAP),
+.BR
 pcap_tstamp_type_name_to_val(3PCAP),
+.BR
 pcap_set_tstamp_precision(3PCAP),
+.BR
 pcap_open_offline_with_tstamp_precision(3PCAP),
+.BR
 pcap_fopen_offline_with_tstamp_precision(3PCAP),
+.BR
 pcap_get_tstamp_precision(3PCAP)
